samples/db.yml

```diff
@@ -183,6 +183,48 @@ definitions:
       id_app:
         type: string
 
+  ThingAllowingBatchWrites:
+    x-db:
+      AllowOverwrites: false
+      AllowPrimaryIndexScan: true
+      AllowBatchWrites: true
+      DynamoDB:
+        KeySchema:
+          - AttributeName: name
+            KeyType: HASH
+          - AttributeName: version
+            KeyType: RANGE
+        GlobalSecondaryIndexes:
```

Comment:
Can we remove the GSI and make this sample simpler? I don't think this block is required for testing `AllowBatchWrites`